ocf_data_sampler/select/time_slice_for_dataset.py

```diff
@@ -51,19 +51,14 @@ def slice_datasets_by_time(
             interval_end=minutes(sat_config.interval_end_minutes),
         )
 
-        # Randomly sample dropout
-        sat_dropout_time = draw_dropout_time(
+        # Apply dropout using the combined function
```

Comment:
```suggestion
        # Apply dropout
```

It's the only function now, so I don't think it needs to be specified

Reply:
I'll update the comment.